composer.json

```diff
@@ -7,7 +7,7 @@
     "php": ">=7.1",
     "google/gax": "^1.0.2",
     "ulrichsg/getopt-php": "^3.2.2",
-    "monolog/monolog": "^1.23.0"
+    "monolog/monolog": "^1.23.0|^2.0"
```

Comment:
Can it be "||" like php_codesniffer below?

Reply:
@fiboknacky Done.